hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/handlers/NodeUpdateHandler.java

```diff
@@ -65,8 +65,9 @@ public void pureChecks(@NonNull final TransactionBody txn) throws PreCheckExcept
         requireNonNull(txn);
         final var op = txn.nodeUpdate();
         validateFalsePreCheck(op.nodeId() < 0, INVALID_NODE_ID);
-        if (op.hasGossipCaCertificate())
+        if (op.hasGossipCaCertificate()) {
```

Comment:
This seems likely to be a merge issue, since this PR does not target DAB. I remember @iwsimon changing this in a recent PR, but I can't find it now. @iwsimon does this look correct?

Reply:
This PR should not change any DAB class. This change just to add {} around the code. Not needed.